devices/MAX1704x/README.MD

```diff
@@ -0,0 +1,24 @@
+# MAX1704x battery gauge
+Library designed for controlling MAX1704x IC.
+
+### Functions
+The binding supports all funcionalities for MAX1704x family. Implemented based on https://github.com/sparkfun/SparkFun_MAX1704x_Fuel_Gauge_Arduino_Library/blob/main/src/SparkFun_MAX1704x_Fuel_Gauge_Arduino_Library.cpp
+
+### Classes
+* Max17043 - class for MAX17043
+* Max17044 - class for MAX17044
+* Max17048 - class for MAX17048
+* Max17049 - class for MAX17049
+* Max1704x - base class for all max IC
+
+## Usage
+
```

Comment:
Can you copy/paste the usage we have in other I2C devices for ESP32 to setup the pins?
IT's a standard thing we've been doing to help in such cases.